src/Mod/PartDesign/Gui/TaskMultiTransformParameters.cpp

```diff
@@ -308,66 +316,85 @@ void TaskMultiTransformParameters::onTransformAddLinearPattern()
     closeSubTask();
     std::string newFeatName = TransformedView->getObject()->getDocument()->getUniqueObjectName("LinearPattern");
     auto pcActiveBody = PartDesignGui::getBody(false);
-    if(!pcActiveBody) return;
+    if (!pcActiveBody)
+        return;
 
     Gui::Command::openCommand(QT_TRANSLATE_NOOP("Command", "Make LinearPattern"));
-    FCMD_OBJ_CMD(pcActiveBody,"newObject('PartDesign::LinearPattern','"<<newFeatName<<"')");
+    FCMD_OBJ_CMD(pcActiveBody, "newObject('PartDesign::LinearPattern','"<<newFeatName<<"')");
     auto Feat = pcActiveBody->getDocument()->getObject(newFeatName.c_str());
+    if (!Feat)
+        return;
     //Gui::Command::updateActive();
     App::DocumentObject* sketch = getSketchObject();
     if (sketch) {
-        FCMD_OBJ_CMD(Feat,"Direction = ("<<Gui::Command::getObjectCmd(sketch)<<",['H_Axis'])");
+        FCMD_OBJ_CMD(Feat, "Direction = ("<<Gui::Command::getObjectCmd(sketch)<<",['H_Axis'])");
     }
     else {
         // set Direction value before filling up the combo box to avoid creating an empty item
         // inside updateUI()
         PartDesign::Body* body = static_cast<PartDesign::Body*>(Part::BodyBase::findBodyOf(getObject()));
         if (body) {
-            FCMD_OBJ_CMD(Feat,"Direction = ("<<Gui::Command::getObjectCmd(body->getOrigin()->getX())<<",[''])");
+            FCMD_OBJ_CMD(Feat, "Direction = ("<<Gui::Command::getObjectCmd(body->getOrigin()->getX())<<",[''])");
         }
     }
 
-    FCMD_OBJ_CMD(Feat,"Length = 100");
-    FCMD_OBJ_CMD(Feat,"Occurrences = 2");
+    FCMD_OBJ_CMD(Feat, "Length = 100");
+    FCMD_OBJ_CMD(Feat, "Occurrences = 2");
 
     finishAdd(newFeatName);
+    // show the new view when no error
+    if (Feat)
+        if (!Feat->isError())
+        TransformedView->getObject()->Visibility.setValue(true);

```

Comment:
Indentation looks off here - did you mean to combine the two if statements?

Reply:
Just a paste mistake. I fixed this now.